ush/python/pygfs/task/aero_emissions.py

```diff
@@ -233,9 +184,9 @@ def open_qfed(files: List[str], out_var_dict: Dict[str, str] = None) -> xr.Datas
                 with xr.open_dataset(f, decode_cf=False).biomass as da:
                     da.name = out_var_dict[input_var]
                     dset_dict[da.name] = da
-            except Exception as ee:
-                logger.exception(f"FATAL ERROR: unable to read dataset {ee}")
-                raise Exception("FATAL ERROR: Unable to read dataset, ABORT!")
+            except Exception as e:
+                logger.exception(f"Unable to read dataset: {f}")
```

Comment:
Even though it may seem a bit redundant with an uncaught exception, I think we want to keep "FATAL ERROR" in the logger (NCO will want to see it).
```suggestion
                logger.exception(f"FATAL ERROR: Unable to read dataset: {f}")
```